policy/modules/system/systemd.if

```diff
@@ -600,6 +600,25 @@ interface(`systemd_log_parse_environment',`
 	typeattribute $1 systemd_log_parse_env_type;
 ')
 
+######################################
+## <summary>
+##   Make the specified type usable as a
+##   systemd user session type.
+## </summary>
+## <param name="domain">
+##   <summary>
+##     Type to be used as a systemd user session type.
+##   </summary>
+## </param>
+#
+interface(`systemd_systemd_user_session',`
```

Comment:
I'm not clear why one would need this except since it that is what `systemd_role()` is for?

Reply:
Please see the use of sister interface systemd_log_parse_environment() in systemd.te...

Reply:
If a new such kind of attribute is introduced, such as "systemd_systemd_user_session" that has been introduced in template systemd_role_template(), then you also need to provide an interface that amkes it possible to actually use such attribute easily in the policy...